src/app/speechtotext/speechtotext.component.ts

```diff
@@ -34,6 +34,11 @@ export class SpeechtotextComponent implements OnInit {
       this.shadowtop = '-102px';
     }
     this.speechRecognition();
+
+    let _setTimer = Observable.timer(1500);
```

Comment:
instead of creating a new timer, you can use the timer in `ngOnInit()` and set message to 'Listening...' there.